source/winVersion.py

```diff
@@ -26,20 +72,26 @@ def __init__(
 			major: int = 0,
 			minor: int = 0,
 			build: int = 0,
+			releaseName: Optional[str] = "",

```

Comment:
This  is quite unusual. The standard usage for `Optional` is for arguments which are set to `None` by default yet can accept a given type. Here however you're setting it into an empty string as a default value and passing `None` when constructing the instance. Can this be simplified to either always using empty  string and not accepting `None` or preferably using `None` as default value for this kwarg?